src/openforms/validations/registry.py

```diff
@@ -26,6 +26,7 @@ class RegisteredValidator:
     verbose_name: str
     callable: ValidatorType
     is_demo_plugin: bool = False
+    is_enabled: bool = True  # TODO always enabled for now
```

Comment:
Since this validator doesn't inherit from any `AbstractBasePlugin` class, this has to be added here explicitly. 

I'm not sure if this is something that still has to be refactored, or if it's intentionally different from the other plugins?

Reply:
Please create an issue to bring this module in line as well

Reply:
Created https://github.com/open-formulieren/open-forms/issues/1149